meta/gist-raw-links.meta.js

```diff
@@ -0,0 +1,21 @@
+// ==UserScript==
+// @name        Gist Raw Links
+// @version     0.2.3
+// @description Add a button that contains a list of gist raw file links
+// @license     MIT
+// @author      Rob Garrison
+// @namespace   https://github.com/Mottie
+// @include     https://gist.github.com/*
+// @run-at      document-idle
+// @grant       GM_addStyle
+// @grant       GM.addStyle
+// @grant       GM_xmlhttpRequest
+// @grant       GM.xmlHttpRequest
+// @connect     api.github.com
+// @connect     assets-cdn.github.com
+// @require     https://greasemonkey.github.io/gm4-polyfill/gm4-polyfill.js?updated=20180103
+// @icon        https://github.githubassets.com/pinned-octocat.svg
+// @updateURL   https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/meta/gist-raw-links.meta.js
+// @downloadURL https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/gist-raw-links.user.js
+// @supportURL  https://github.com/Mottie/GitHub-userscripts/issues
+// ==/UserScript==
```

Comment:
```diff
// ==UserScript==
-// @name        Gist Raw Links
// @version     0.2.3
-// @description Add a button that contains a list of gist raw file links
-// @license     MIT
-// @author      Rob Garrison
-// @namespace   https://github.com/Mottie
-// @include     https://gist.github.com/*
-// @run-at      document-idle
-// @grant       GM_addStyle
-// @grant       GM.addStyle
-// @grant       GM_xmlhttpRequest
-// @grant       GM.xmlHttpRequest
-// @connect     api.github.com
-// @connect     assets-cdn.github.com
-// @require     https://greasemonkey.github.io/gm4-polyfill/gm4-polyfill.js?updated=20180103
-// @icon        https://github.githubassets.com/pinned-octocat.svg
-// @updateURL   https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/meta/gist-raw-links.meta.js
-// @downloadURL https://raw.githubusercontent.com/Mottie/GitHub-userscripts/master/gist-raw-links.user.js
-// @supportURL  https://github.com/Mottie/GitHub-userscripts/issues
// ==/UserScript==
```
Considering the scope of this PR, would that work? :-)

Reply:
I think `updateURL` 	and `downloadURL` might be required depending on how it was implemented on the client’s.
Others are not required probably.

Reply:
I think I should pr again with the files.